Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Storefront.m

```diff
@@ -126,10 +126,20 @@ - (NSOperation *)getProductsByIds:(NSArray *)productIds completion:(BUYDataProdu
 
 - (NSOperation *)getProductTagsPage:(NSUInteger)page completion:(BUYDataTagsListBlock)block
 {
-	NSURL *url  = [self urlForProductTagsWithParameters:@{
-														  @"limit" : @(self.pageSize),
-														  @"page"  : @(page),
-														  }];
+	return [self getProductTagsInCollection:@"" page:page completion:block];
+}
+
+- (NSOperation *)getProductTagsInCollection:(NSString *)collectionId page:(NSUInteger)page completion:(BUYDataTagsListBlock)block
+{
+	NSMutableDictionary *params = @{
+									@"limit" : @(self.pageSize),
+									@"page"  : @(page),
+									}.mutableCopy;
+	if (collectionId.length) {
```

Comment:
`collectionId` is only privately optional, as this line is/should only be exercised by line 129 ( @bgulanowski )